docs/api/libraries/polymer/index.md

```diff
@@ -31,20 +31,22 @@ These custom elements fire [polymer](https://polymer-library.polymer-project.org
 
 ```ts wcd 9kmJUbA735YuF4HRBzO7 src/Hello.ts
 import { PolymerElement, html } from '@polymer/polymer';
+import '@apollo-elements/components/apollo-client';
 import '@apollo-elements/polymer/apollo-query';
 
 class HelloQueryElement extends PolymerElement {
   static get template() {
     return html`
-      <apollo-query data="{%raw%}{{data}}{%endraw%}">
-        <script type="application/graphql">
-          query HelloQuery($name: String, $greeting: String) {
-            hello(name: $name, greeting: $greeting) {
-              name
-              greeting
-            }
-          }
-        </script>
+      <apollo-client
+        uri="/graphql"
+        client="{%raw%}{{client}}{%endraw%}">
```

Comment:
The intended use is to bind to a DOM tree,

But this would be good to add as a second snippet to show that two-way binding is available as well